### ISSUE TITLE
terminate called after throwing an instance of 'std::out_of_range'

### ISSUE THREAD

**Issue by MaximArtemev** [philiprodrigues/waveform-tools#1]
Running this line

`extract_larsoft_waveforms -i ../file_name.root -o out_file --numpy`

results in this exception 

```
terminate called after throwing an instance of 'std::out_of_range'
what():  basic_string::substr: __pos (which is 18446744073709551615) > this->size() (which is 8)
```

Full output: 

```
Successfully opened file ../file_name.root
Event run: 20000004 subRun: 0 event: 1
%MSG-d ProductID: 
Product created with id: [110618289] from canonical product name: "raw::RawDigits_daq__GenieGen."
%MSG
%MSG-d ProductID: 
Product created with id: [4150374642] from canonical product name: "raw::RawDigits_daq__G4."
%MSG
%MSG-d ProductID: 
Product created with id: [1541183742] from canonical product name: "raw::RawDigits_daq__NoZSDetsim."
%MSG
terminate called after throwing an instance of 'std::out_of_range'
  what():  basic_string::substr: __pos (which is 18446744073709551615) > this->size() (which is 8)

```

**Comment by philiprodrigues** [comment]
Do you have the input file on lxplus? Could you tell me the exact path? I'll try to reproduce the problem

**Comment by MaximArtemev** [comment]
I'm using MC11 files on lxplus. You can get those on https://cernbox.cern.ch/index.php/s/oPleDv7grRLbbeC

**Comment by philiprodrigues** [comment]
Fixed by commit 8201fb4 - thanks for reporting! (Or for a workaround, just add an extension to your output file name, eg .npy)
